.github/workflows/bench.yml

```diff
@@ -161,21 +161,23 @@ jobs:
   wtelegramclient:
     if: github.event.inputs.library == '05_wtelegramclient' || github.event.inputs.library == 'All'
     runs-on: ubuntu-latest
+    permissions:
+      actions: write
     env:
       api_id: ${{ secrets.API_ID }}
       api_hash: ${{ secrets.API_HASH }}
       bot_token: ${{ secrets.BOT_TOKEN_05 }}
-      MESSAGE_LINK: ${{ secrets.MESSAGE_LINK }}
-      CHAT_ID: ${{ secrets.CHAT_ID }}
-      session_pathname: ./WTelegram.session
+      message_link: ${{ secrets.MESSAGE_LINK }}
+      chat_id: ${{ secrets.CHAT_ID }}
+      session_pathname: WTelegram.session
     steps:
       - uses: actions/checkout@v4
 
       - uses: actions/setup-dotnet@v4
         with:
           dotnet-version: "8.x"
 
-      - uses: actions/cache@v4
+      - uses: actions/cache/restore@v4
         with:
           path: 05_wtelegramclient/WTelegram.session
           key: wtelegramclient
```

Comment:
```suggestion
      - uses: actions/cache@v4
        with:
          path: 05_wtelegramclient/WTelegram.session
          key: wtelegramclient-${{ hashFiles('05_wtelegramclient/WTelegram.session') }})
```

Reply:
I think public repositories have good limits for actions. And why would you doubt it working?